src/checks_linux.go

```diff
@@ -46,6 +46,18 @@ func processCheck(check *check, checkType string, arg1 string, arg2 string, arg3
 		}
 		result, err := packageVersion(arg1, arg2)
 		return err == nil && !result
+	case "KernelVersion":
+		if check.Message == "" {
+			check.Message = "Kernel has been updated to version " + arg1
+		}
+		result, err := kernelVersion(arg1)
+		return err == nil && result
+	case "KernelVersionNot":
+		if check.Message == "" {
+			check.Message = "Kernel has not been updated to version " + arg1
```

Comment:
May or may not be an update, this is really nitpicky but it might be more accurate to say "Kernel is version X" and "Kernel is not version X"